### PR TITLE
Ladders landing

### PR DESCRIPTION
**Pull request by SavageWilliam**
#7 #6  creates page after login, with current ladders and option to create new ladder. Satisfies user stories in issues referenced. 

@des-des - unable to add you as reviewer currently. 

@nogainbar currently we have some structure but the design needs working on quite a lot.

### CODE REVIEW

**Comment by bradreeder** [comment]
@SavageWilliam have a feeling it's because he has an invite pending into the organisation. should resolve shortly.

**Comment by bradreeder** [comment]
Resolved ^^;

File: css/ladders-landing.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  height: 60vh;
+  position: fixed;
+  z-index: 100;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  background: white;
+  box-shadow: 0 0 60px 10px rgba(0, 0, 0, 0.9);
+}
+.closed {
```
Comment (des-des) [inline css/ladders-landing.css]: is this the correct classname? should it be modal_closed?

File: modal.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+var modal = document.querySelector("#modal");
+var modalOverlay = document.querySelector("#modal-overlay");
+var closeButton = document.querySelector("#close-button");
+var openButton = document.querySelector("#open-button");
+
+closeButton.addEventListener("click", function() {
+  modal.classList.toggle("closed");
```
Comment (des-des) [inline modal.js]: yup here as well, `modal_closed`?

File: css/ladders-landing.css (hunk 1 above)
Comment (SavageWilliam) [inline css/ladders-landing.css]: because the class just sets display to none, is it alright to keep it as closed?

Comment (des-des) [inline css/ladders-landing.css]: If you are happy I am happy.

But one of the ideas of BEM is to stop having classnames like `closed` around.

Ie see this https://jqueryui.com/accordion/.

See that here we might use the name: accordian_closed for a folded down element, as closed has a different meaning in different contexts.

Ie when using bem you do not have the classname `red`.. but you might have the classname `button_disabled` to make a button `red` or something.

Comment (des-des) [inline css/ladders-landing.css]: I am happy to merge this as minor, if you disagree with me

**Comment by des-des** [comment]
Is this ready?

**Comment by jsms90** [comment]
@des-des We were just waiting for @nogainbar

**Comment by jsms90** [comment]
Removing Noga, since this version is not indicative of our final design.

@des-des Can you merge please? :+1:

**Pull request merged**